packages/pangea-multipass/pangea_multipass/sources/jira/jira.py

```diff
@@ -193,6 +207,26 @@ def _request(auth: JiraAuth, path: str, params: dict = {}) -> dict:
         response.raise_for_status()
         return response.json()
 
+    @staticmethod
+    def _post(auth: JiraAuth, path: str, body: dict = {}) -> dict:
+        headers = {
+            "Accept": "application/json",
+            "Content-Type": "application/json"
+        }
+
+        basic_auth = HTTPBasicAuth(auth.email, auth.token)
+
+        response = requests.request(
+            "POST",
+            f"https://{auth.url}{path}",
+            data=json.dumps(body),
```

Comment:
I believe this can be:

```suggestion
            json=body,
```

See the last part of <https://docs.python-requests.org/en/latest/user/quickstart/#more-complicated-post-requests>

Reply:
fixed!